source/playbooks/SC/ssmdocs/SC_IAM.22.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { Construct } from 'constructs';
+import { ControlRunbookDocument } from './control_runbook';
+import { PlaybookProps } from '../lib/control_runbooks-construct';
+import { RevokeUnusedIAMUserCredentialsDocument } from './SC_IAM.8';
+
+export function createControlRunbook(stage: Construct, id: string, props: PlaybookProps): ControlRunbookDocument {
+  return new RevokeUnusedIAMUserCredentialsDocument(stage, id, {
+    ...props,
+    controlId: 'IAM.22',
+    parameterToPass: '45',
```

Comment:
could this be named something more descriptive? I'm not entirely sure the purpose of the remediation but maybe something like "LifetimeDaysExceeding"?

Reply:
In this instance it makes sense for there to be a generic name because this props struct is used by different remediations. There's probably a more elegant way of specifying something like this, where the implementation only varies by a value or two across different controls/standards, but that's not an abstraction that this was originally designed for.